adding_species_manuscript.tex

```diff
@@ -390,7 +390,7 @@ \section*{Additions to \stdpopsim}
 two flowering plants including a crop (\textit{Helianthus annuus}),
 an algae (\textit{Chlamydomonas reinhardtii}),
 two bacteria,
-four primates and a common mammalian associate of primates (\textit{Canis familiaris}).
+four primates and a common mammalian associate of humans (\textit{Canis familiaris}).
```

Comment:
I think that 'primates' was used here on purpose, as a little wink to the attentive reader

Reply:
Agreed. Although "primates" is less precise (dogs aren't exactly a common associate of chimps or lemurs, except as a potential predator), so someone without a sense of humor could potentially take issue. (Another thought - could this trip up someone whose native language is not English?)

Reply:
I guess we should avoid the inside joke here and switch to the more straightforward 'humans'